### ISSUE TITLE
Domain expired

### ISSUE THREAD

**Issue by jocelyn-stericker** [trevordixon/LilyBin#87]
lilybin.com is expired and redirects to http://domain-registrar.storage.googleapis.com/expired.html?lilybin.com

**Comment by TimothyGu** [comment]
@trevordixon?

**Comment by trevordixon** [comment]
Renewed. I'm thinking of redirecting to https://www.hacklily.org someday.
Is there anything more desirable about lilybin at this point?

**Comment by jocelyn-stericker** [comment]
Lilybin saves to Dropbox while Hacklily saves to GitHub. Some people may prefer using Dropbox. There is also the matter of all the files saved to Lilybin itself. You probably would want to make sure there’s a way for people to continue having access to those.

I’d love the redirect, of course! Do you have data about how often people save to Dropbox? If you do want to make lilybin redirect, we should figure out a way of making sure the songs saved to lilybin aren’t lost!

**Comment by jocelyn-stericker** [comment]
@TimothyGu / @trevordixon, is merging something you want to discuss more in detail? If you're interested, I'd be happy to do anything I can to make the transition smooth.

**Comment by trevordixon** [comment]
@jnetterf Yeah, I'd probably like to redirect to you at some point. Let's talk about it soon.

**Comment by chrispolley** [comment]
The domain has expired again.  While I understand the line of conversation above, there needs to be a way to extract work that was saved in lilybin, plus a notice to users that the service will be going away if that is the direction you want to go.

Thanks!

**Comment by ftherese** [comment]
yes! please help!  we have a whole grade school that were saving their work as lilybin urls!  We'll do whatever we can to get this back working as it was through the end of the school year.

**Comment by TimothyGu** [comment]
Before @trevordixon is able to renew the domain, you can access the website through http://lilybin.timothygu.me/. It has all the saved scores, etc.
